vizro-core/docs/pages/user-guides/table.md

```diff
@@ -534,5 +534,113 @@ If the available arguments are not sufficient, there is always the option to cre
 
 ## Add title, header, footer
 
-To enhance your `Table` or `AgGrid` with a title, header, or footer to provide additional context or
-descriptions, refer to our detailed user guide on [title, header, and footer](title-header-footer.md).
+The [`Table`][vizro.models.Table] and the [`AgGrid`][vizro.models.AgGrid] models accept a `title`, `header` and
+`footer` argument. This is useful for providing context to the data being displayed, or for adding a description of
+the data.
+
+- **title**: Displayed as an [H3 header](https://dash.plotly.com/dash-html-components/h3), useful for summarizing the main topic or insight of the component.
+- **header**: Accepts Markdown text, ideal for additional descriptions, subtitles, or detailed data insights.
+- **footer**: Accepts Markdown text, commonly used for citing data sources, providing information on the last update, or adding disclaimers.
```

Comment:
We prefer markdown over Markdown as per Vale https://github.com/mckinsey/vizro/issues/733

Reply:
I am going to leave this "mistake" in so we have an easy issue where people could potentially contribute to the GHC